_template/.github/workflows/main.yml

```diff
@@ -18,15 +18,14 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.19.x]
         os: [ubuntu-latest]
```

Comment:
I guess we can also remove this then. And we should probably upgrade to 1.20 now